### ISSUE TITLE
"Expected SQL item" bug when using edge field with order by computed column on mutation payload

### ISSUE THREAD

**Issue by benjie** [graphile/graphile-engine#740]
### Summary

The edge field on mutation payloads throws an error when you attempt to use it with an orderBy that includes a computed column (with `@sortable` smart tag).

### Steps to reproduce

See: https://github.com/pszafer/postgraphile_errors/tree/delete_error

### Expected results

No error, correct edge/cursor returned.

### Actual results

```
"Expected SQL item, instead received '({\n        queryBuilder\n      }) => sql.fragment`(${sql.identifier(proc.namespaceName, proc.name)}(${queryBuilder.getTableAlias()}))`'.",
```

### Additional context

PostGraphile v4.11.0.

### Possible Solution

The error lies here:

https://github.com/graphile/graphile-engine/blob/b715f6d1be7dd19e39830b4329fa3c758e5ae91b/packages/graphile-build-pg/src/plugins/PgMutationPayloadEdgePlugin.js#L170-L174

Specifically we've assumed if it's not a string then it's an SQL node, but that's not the case - it can also be a thunk.

Normally when ordering we call `queryBuilder.orderBy` with a value like the above:

https://github.com/graphile/graphile-engine/blob/1bc8cfefdab7a61fd7ad287bcdff66298352e308/packages/graphile-build-pg/src/plugins/PgConnectionArgOrderBy.js#L165

QueryBuilder automatically calls thunks when generating the compiled data:

https://github.com/graphile/graphile-engine/blob/1bc8cfefdab7a61fd7ad287bcdff66298352e308/packages/graphile-build-pg/src/QueryBuilder.js#L870-L875

which we've not done in the edge plugin. This is done because it passes a reference to the context: 

https://github.com/graphile/graphile-engine/blob/1bc8cfefdab7a61fd7ad287bcdff66298352e308/packages/graphile-build-pg/src/QueryBuilder.js#L138-L140

which includes the QueryBuilder so that orders may use things like the query builder's alias whilst building their fragments.

The PgMutationPayloadEdgePlugin does have access to a QueryBuilder so we should be able to fix this by changing the code above to something along the lines of:

```js
const expr = isString(col)
  ? sql.fragment`${queryBuilder.getTableAlias()}.${sql.identifier(col)}`
  : typeof col === 'function'
  ? col({ queryBuilder })
  : col;
```

**Comment by benjie** [comment]
Oh, this was reported by @pszafer via Discord: https://discord.com/channels/489127045289476126/498852330754801666/842725956841766942

**Comment by benjie** [comment]
Fixed in #741
